src/php/Command/CommandFacade.php

```diff
@@ -51,21 +36,12 @@ public function executeRunCommand(string $fileOrPath): void
 
     /**
      * @param list<string> $paths
-     *
-     * @throws CompilerException
-     * @throws CompiledCodeIsMalformedException
-     * @throws FileException
-     * @throws CannotFindAnyTestsException
      */
     public function executeTestCommand(array $paths): void
     {
-        $result = $this->commandFactory
+        $this->commandFactory
```

Comment:
As far as I can see this isn't sending no output codes (SUCCESS_CODE, FAILED_CODE) anymore? This is required for the CI Pipeline to indicate that the tests failed.

Reply:
Ups, I didn't know this. Ok, I will rollback this as how it was before 🪂